packaging/debian/copyright

```diff
@@ -0,0 +1,35 @@
+Upstream Homepage:
+    https://www.gen2vdr.de/wirbel/w_scan/index2.html
+
+Upstream Author:
+    Winfried Koehler <w_scan AT gmx-topmail DOT de>
+
+Debian Maintainers:
+    Tobias Grimm <etobi@debian.org>
```

Comment:
I see you copied these files from the "w-scan" package, but you're the Debian package maintainer know, so replace Tobias' info with yours.